webclient/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
check if there's a way of having the root package.json call this one in the npm documentation.